core/modules/layout/layout.api.php

```diff
@@ -335,6 +336,98 @@ function hook_layout_presave(Layout $layout) {
   }
 }
 
+/**
+ * Perform alterations to the list of layouts that match the path of a router
+ * item.
+ *
+ * Modules can implement this hook to allow a given layout to be used for
+ *  multiple or different paths than its defined path.
+ *
+ * This hook is called on every page load; your implementation should quickly
+ * and efficiently determine if it is applicable and return if not.
+ *
+ * If a layout's path includes positioned contexts (e.g., 'node/%') and you
+ * apply the layout to a different path, you may need to set the context
+ * data beyond the default setting done by the layout module. The default
+ * setting happens after invocations of this hook, taking care of any contexts
+ * that were not yet set.
+ *
+ * If you substitute a layout whose placeholder(s) are in a different position
+ * from the path in the router item and you will rely on the system to set the
+ * context, you should set the position of the context(s) in the layout to the
+ * position in the router. Example: the path in the router item is
+ * mymodule/node/% and you wish to use the layout with path node/%. Then you
+ * would find the context at position 2 in the router item and set its
+ *  Context::position variable to 2, so that the layout with path node/% looks
+ * at position 2 to set the context for that placeholder.
+ *
+ * @param Layout[] $layouts
+ *   The list of layouts that should be considered for the path
+ *   $router_item['path']. Note that the final selection of layout will be based
+ *   on contexts and visibility conditions.
+ * @param array $router_item
+ *   The router item for the path. In addition to obtaining the system path at
+ *   $router_item['path'], you can use other elements of $router_item. For
+ *   example, $router_item['original_map'] contains an exploded version of the
+ *   normal path.
+ *
+ * @since 1.20.0 (?)
```

Comment:
```suggestion
 * @since 1.20.0
```

Let's go ahead and assume this will make it.

Reply:
OK. Fixed in an updated PR.